Dockerfile

```diff
@@ -11,9 +11,9 @@ COPY ./requirements.txt /requirements.txt
 
 RUN pip3 install --upgrade pip
 
-RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-auth.git
+RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-auth.git@ad3dab5
 
-RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-rudra#egg=rudra
+RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-rudra.git@e9a9239
```

Comment:
You must have `egg=rudra` argument, otherwise it will take the complete codebase instead of just https://github.com/fabric8-analytics/fabric8-analytics-rudra/tree/master/rudra.

Same comment for the rest of the changes as well.